### ISSUE TITLE
link-preview-js is no longer supported

### ISSUE THREAD

**Issue by NicholasIoanJones** [maherzaidoune/react-native-url-preview#28]
do you have a replacement plan now that https://github.com/ospfranco/link-preview-js is now longer maintained?

**Comment by maherzaidoune** [comment]
> do you have a replacement plan now that https://github.com/ospfranco/link-preview-js is now longer maintained?

not really, no clear plan yet, for now, I'm forking https://github.com/ospfranco/link-preview-js and try to keep it updated
